provisioner_test.go

```diff
@@ -119,3 +119,44 @@ func TestNewDisembedderProvisioner(t *testing.T) {
 	assert.NoError(t, err)
 	testProvisionerSuccessful(t, *p, "linux", "amd64", DefaultVersionAstilectron, DefaultVersionElectron)
 }
+
+func TestZipShouldRemove(t *testing.T) {
+	var o = Options{
+		DataDirectoryPath: mockedTempPath(),
+	}
+
+	// Make sure the test directory doesn't exist.
+	if err := os.RemoveAll(o.DataDirectoryPath); err != nil && !os.IsNotExist(err) {
+		t.FailNow()
+	}
+	defer os.RemoveAll(o.DataDirectoryPath)
+
+	a, err := New(&testingHelper.StdLogger{T: t}, o)
```

Comment:
Here you need to use [this function](https://github.com/asticode/go-astikit/blob/master/logger.go#L183)

Reply:
done. https://github.com/asticode/go-astilectron/pull/358/commits/79222481c3e714efa25ddb81c4325948f97708b5#diff-f5c6513a93e0d3a0d10fb30654a66c3ab4b7f05ed5b4dd9dd10ef379a14496adR134